### PR TITLE
Added install/contribution instructions to readme

### PR DESCRIPTION
**Pull request by alexgleason**
Some people are asking how to install this (#20). And some people are having trouble installing it (#5, #9).

These changes should help address that, while keeping it clear that this package is in an early stage and not ready for the end-user. I've created a "contributing" section with install instruction for contributors. It should help people install the package now, and the instructions will continue to be relevant once this package is in apm.

The intention is the "install" section will be replaced with apm instructions once the package is released.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,27 @@
...
+## Usage
+Activate with <kbd>Ctrl</kbd>-<kbd>Alt</kbd>-<kbd>O</kbd> or through the command panel with `Activate Power Mode: Toggle`. In
+this early version it only affects the current tab. To run it in a new tab you need to run `Window: Reload` first.
+
+## Contributing
+If you haven't already done so, install [Node.js](https://nodejs.org/).
+
+Once Node.js is installed, place this package in your `~/.atom/packages/` directory. It's recommended that you use git to clone this package into that directory, as follows:
+
+    git clone https://github.com/JoelBesada/activate-power-mode.git ~/.atom/packages/activate-power-mode
```
Comment (huanzhang) [inline README.md]: 1) fork first, and then clone from contributor's codebase. here is the github official guide: https://guides.github.com/activities/contributing-to-open-source
2) according to https://atom.io/docs/v1.2.4/hacking-atom-package-word-count, cloning the code to the developer's workspace and link to ~/.atom/packages or ~/.atom/dev/packages is a better choice. On thing to note is if link the source code to ~/.atom/dev/packages, we need enter dev mode with `$ atom -d` to enable the plugin.

**Comment by alexgleason** [comment]
@huanzhang Thanks for pointing those out!

I agree that there should be information about forking in there. Although I'm tempted to ask the user to run `git remote set-url origin /users/fork/path.git` after the fact, that way we don't discourage anyone too early. And I think we should also include a link to the Github contributing guidelines.

I'm curious to know more about why a symlink is better. The link you shared says this:

>You can run the generator by invoking the command palette and searching for "Generate Package". A dialog will appear asking you to name your new project. Atom will then create that directory and fill it out with a skeleton project **and link it into your .atom directory** so it’s loaded when you launch your editor next time.

This is describing the behavior of an Atom feature, but it doesn't say it's better, or explain why they made that choice in this context. Unless there's something I don't know, cloning directly in is equally valid. I think they made the choice here so the location of the dev package is more obvious to the end-user.

IMO this PR can be merged, but I agree that we should add more info about forking and contributing to Github later.

**Comment by huanzhang** [comment]
@alexgleason thanks for your quick reply for my comments.

For 1), I agree with you that we can add more contributing info later.
However, for 2), I still think that developing from ~/.atom/pacages/active-power-mode does not work for me. I would prefer to gather various projects to one folder something like ~/Code or ~/Workspace, here are my two reasons:
1. it's kind of data classification
2. easy to find the project with finder/ide or editor's tree view/ls
3. neither hid with .atom nor store in deep folders

**Comment by JoelBesada** [comment]
Thanks for this! One note though, the package has actually been available through `apm` from the beginning https://atom.io/packages/activate-power-mode, I guess I just haven't made that clear.

**Comment by alexgleason** [comment]
This has been mostly fixed with the latest changes to the readme. Closing.

**Pull request closed without merging**